istioctl/pkg/writer/pilot/status_test.go

```diff
@@ -150,12 +150,12 @@ func statusInput1() []v2.SyncStatus {
 		{
 			ProxyID:         "proxy1",
 			ProxyVersion:    "1.0",
-			ClusterSent:     "2009-11-10 23:00:00 +0000 UTC m=+0.000000001",
-			ClusterAcked:    "2009-11-10 22:00:00 +0000 UTC m=+0.000000001",
-			ListenerSent:    "2009-11-10 23:00:00 +0000 UTC m=+0.000000001",
-			ListenerAcked:   "2009-11-10 23:00:00 +0000 UTC m=+0.000000001",
-			EndpointSent:    "2009-11-10 23:00:00 +0000 UTC m=+0.000000001",
-			EndpointAcked:   "2009-11-10 23:00:00 +0000 UTC m=+0.000000001",
+			ClusterSent:     "2018-09-17T01:52:44.525391Z",
```

Comment:
Probably should have been clearer I didn't mean strip it off the input because that would require wider changes to Pilot. I meant split the string on ` m=+` in the istioctl code and discard the second half.